mojave/structures/ruins.dm

```diff
@@ -0,0 +1,89 @@
+#define RUIN_BODIES_LEFT list("coupe-left" = 10, "muscle-left" = 3, "sport-left" = 3, "van-left" = 1)
+#define RUIN_BODIES_RIGHT list("coupe-right" = 10, "muscle-right" = 3, "sport-right" = 3, "van-right" = 1)
+
+/////////////////////////////////////////////////////////////
+////////////// MOJAVE SUN RUINS/TRASH ///////////////////////
+/////////////////////////////////////////////////////////////
+
+///Place here semi-functional structures that are basically for scrapping/joonking///
+
+//Cars//
+
+/obj/structure/ms13/vehicle
```

Comment:
```suggestion
/obj/structure/ms13/vehicle_ruin
```
I would consider repathing these to something like this, to indicate that they're ruined and not to be confused with `obj/vehicle` (which we might use eventually, if some nut makes us drivable vehicles).